drivers/virtualbox/virtualbox.go

```diff
@@ -62,17 +65,24 @@ type Driver struct {
 // NewDriver creates a new VirtualBox driver with default settings.
 func NewDriver(hostName, storePath string) *Driver {
 	return &Driver{
-		VBoxManager: NewVBoxManager(),
-		BaseDriver: &drivers.BaseDriver{
-			MachineName: hostName,
-			StorePath:   storePath,
-		},
+		VBoxManager:         NewVBoxManager(),
```

Comment:
Will unmarshaling hosts from old versions work OK with these new additions?  I think it might be OK, because the newly added objects are not pointers (consequently they will be initialized to an empty struct, not `nil`), but we should make sure to test thoroughly.